README

```diff
@@ -25,24 +25,33 @@ For more information and details on how to contribute to `htop` visit [htop.dev]
 
 ## Build instructions
 
+### prerequisite
 This program is distributed as a standard GNU autotools-based package.
 
 Compiling `htop` requires the header files for `ncurses` (libncursesw*-dev). Install these and other required packages for C development from your package manager.
 
-Then, when compiling from a [release tarball](https://bintray.com/htop/source/htop), run:
+**Debian/ubuntu**
+~~~ Bash
+sudo apt install libncursesw5-dev autotools-dev autoconf
+~~~
+
+### compiling from tarball:
+
+When compiling from a [release tarball](https://bintray.com/htop/source/htop), run:
 
 ~~~ shell
 ./configure && make
 ~~~
 
-Alternatively, for compiling sources downloaded from the Git repository (`git clone` or downloads from [Github releases](https://github.com/htop-dev/htop/releases/)),
-install the header files for `ncurses` (libncursesw*-dev) and other required development packages from your distribution's package manager. Then run:
+### compiling from source:
+Alternatively, for compiling sources downloaded from the Git repository (`git clone` or downloads from [Github releases](https://github.com/htop-dev/htop/releases/)), then run:
 
 ~~~ shell
 ./autogen.sh && ./configure && make
 ~~~
```

Comment:
This is no longer up-to-date, as the source tarballs will likely not ship the `configure` script anymore (cf. #618), thus the full cycle is necessary. Also you might want to include the call to `make` to actually compile the software. ;-)